src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/RealTimeLength/RealTimeLength.cpp

```diff
@@ -0,0 +1,51 @@
+#include "RealTimeLength.h"
+#include <iostream>
+using namespace std;
+
+RealTimeLength::RealTimeLength(int* Asindex, double* Astack, int* Bsindex, double* Bstack){
+    sindex=0;                       //temporary stackindex reset
+	lastLength=0.0;                 //last measure reset
+	difference=SENSOR_DISTANCE;     //set infra sensor distance
+	sstack[0]=Astack;               //set pointer to A infra's datastack
+	sstack[1]=Bstack;               //set pointer to B infra's datastack
+	ssindex[0]=Asindex;             //set pointer to A infra's stackindex
+	ssindex[1]=Bsindex;             //set pointer to B infra's stackindex
+}
+/*bool RealTimeLength::Update(){

```

Comment:
Absolutely useless (to commit code in comments), since from the commit history you could retrieve this method if needed.